bundle/edu.gemini.p2checker/src/test/scala/edu/gemini/p2checker/rules/GmosSDtaXRuleTest.scala

```diff
@@ -71,6 +78,11 @@ class GmosSDtaXRuleTest extends AbstractRuleTest {
 
   private def expectFailure2(): Unit = expectFailure("Ybin=2")
```

Comment:
I'm trying not to modify much of what's already written and just add new code necessary to fix the bug. I still don't feel familiar enough with the code to renames things. However this concrete case looks trivial enough.